### PR TITLE
(Fix): Redirect and styles issues

### PR DESCRIPTION
**Pull request by AVRahul**
Signed-off-by: AVRahul <a.v.rahul11@gmail.com>

**Issues:**
- Github button redirects to the same tab.
- Blog navbar selected tag color fix for mobile view.
- Installation section button styles are overlapping.

### CODE REVIEW

**Review by satyapriyamishra222 — APPROVED** [review]

File: website/src/pages/Home/styles.ts [hunk 1]
```diff
@@ -293,14 +293,23 @@ const useStyles = makeStyles((theme) => ({
...
+    overrides: {
+        '.MuiButton-contained': {
+            background: theme.palette.background.paper,
+            width: '184px',
+        },
+        '.MuiButton-root': {
+            borderRadius: '8px',
+        }
+    },
     installationButton: {
```
Comment (rakeshPRaghu) [inline website/src/pages/Home/styles.ts:305]:
```suggestion
    installationButton: { 
         '&.MuiButton-contained' : { 
```
this should work, PTAL

Comment (AVRahul) [reply website/src/pages/Home/styles.ts:305]: Thanks @rakeshPRaghu it worked!!

File: website/src/pages/Home/styles.ts [hunk 2]
```diff
@@ -293,26 +293,39 @@ const useStyles = makeStyles((theme) => ({
...
     },
+    overrides: {
+        '.MuiButton-contained': {
+            background: theme.palette.background.paper,
+            width: '184px',
+        },
+        '.MuiButton-root': {
+            borderRadius: '8px',
+        }
+    },
```
Comment (rakeshPRaghu) [inline website/src/pages/Home/styles.ts:304]: why do we need this?

Comment (AVRahul) [reply website/src/pages/Home/styles.ts]: Not required, removed it.

File: website/src/components/Header/index.tsx [hunk 3]
```diff
@@ -32,8 +32,8 @@ const Header: React.FC = () => {
     const displaySocialLinks = () => {
           return (
               <div className={classes.socialIconsWrapper}>
-                    <Link href={EXTERNAL_LINKS.OPENEBS_GITHUB_REPO}>
-                        <img loading="lazy" src="/images/logos/githubLogo.svg" className={classes.socialIcon} alt={t('generic.github')}/>
+                    <Link href={EXTERNAL_LINKS.OPENEBS_GITHUB_REPO} target="_blank" rel="noopener">
+                        <img loading="lazy" src="../images/logos/githubLogo.svg" className={classes.socialIcon} alt={t('generic.github')}></img>
```
Comment (satyapriyamishra222) [inline website/src/components/Header/index.tsx:36]: Why a closing tag for img ?

Comment (rakeshPRaghu) [inline website/src/components/Header/index.tsx:36]: if you rebased PTAL once more, i think there was some change from @Pallavi-PH here!

Comment (AVRahul) [reply website/src/components/Header/index.tsx:36]: yes, I guess some rebase issues, checking it..

Comment (AVRahul) [reply website/src/components/Header/index.tsx:36]: I guess some rebase issues caused this, will check and push again.

Comment (AVRahul) [reply website/src/components/Header/index.tsx:36]: Sure will remove the closing tag.

Comment (AVRahul) [reply website/src/components/Header/index.tsx]: There was some rebase issue, have resolved them.

**Review by rakeshPRaghu — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Review by Pallavi-PH — APPROVED** [review]

**Pull request merged**